### PR TITLE
is_type_of for stacks

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: tools/stacks/macros.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@ Stacks was developed by Julian Catchen with contributions from Angel Amores, Pau
 
 `Stacks website <http://creskolab.uoregon.edu/stacks/>`_
```
Comment (nsoranzo) [inline tools/stacks/macros.xml]: http://catchenlab.life.illinois.edu/stacks/

**Comment by abretaud** [comment]
+1 for me, I guess we can merge it?

**Comment by mvdbeek** [comment]
Nice, the retesting worked. Yeah, feel free to merge.

**Comment by abretaud** [comment]
Great, thanks!

**Pull request merged**